.github/workflows/pr-automation.yml

```diff
@@ -73,13 +73,27 @@ jobs:
             const branch = context.payload.pull_request.base.ref
             
             if (milestones.data.length == 0) {
+              await github.rest.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                body: '@' + context.payload.pull_request.merged_by.login +
+                ' There are no open milestones ... please create one'
+              });
               throw new Error('There are no open milestones ... please create one')
             }
 
             var milestone;
             if (milestones.data.length > 1) {
               milestone = milestones.data.find(({description}) => description?.includes('branch: ' + branch));
               if (!milestone) {
+                await github.rest.issues.createComment({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  issue_number: context.issue.number,
+                  body: '@' + context.payload.pull_request.merged_by.login +
+                  ' There are more then oen open milestones ... please add a "branch: ' + branch + '" to description in one milestone'
```

Comment:
```suggestion
                  ' The PR can't be associated to a milestone, because there are multiple open milestones. Please add the text "branch: ' + branch + '" to the description to the milestone where this PR belongs to.'
```